src/SimCell.h

```diff
@@ -14,11 +14,17 @@ class SimCell
 	void SetCellState(bool state);
 	int GetRespawnCounter();
 	int GetKillCounter();
+	void IncreaseMyLifeNeighbors();
+	void ResetLifeNeighborsCounter();
+{
 protected:
 private:
 	bool isAlive;
 	int killCounter;
 	int respawnCounter;
+	// in the end of analize phase this variable decidade compare with simulation rules to still live or dead
+	// dead neighbours is 9 - life neighbours (not counting cell on edge - solution in progress)
+	int myLiveNeighbors;
```

Comment:
Uninitialized and has garbage value at the beginning of simulation - should be set in the ctor.